packages/tasks/src/model-libraries.ts

```diff
@@ -469,6 +469,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path_extension:"nemo" OR path:"model_config.yaml"`,
 	},
+	open_oasis: {
+		prettyLabel: "open-oasis",
+		repoName: "open-oasis",
+		repoUrl: "https://github.com/etched-ai/open-oasis",
+		filter: true,
+		countDownloads: `path_extension:"pt"`,
```

Comment:
```suggestion
		countDownloads: `path:"oasis500m.pt"`,
```

Better to count on a specific file otherwise downloads will be double counted (since there is `oasis500m.pt` and `vit-l-20.pt`)

Reply:
Agreed!